index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <!--
+    Hello Camper!
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+    - The freeCodeCamp Team
+    -->
+  <script src="js/index.js"></script>
+  <div class="head">
+    <h1>FreecodeCamp Survey Form</h1>
+    <p>Thank you for taking the time to help us improve the platform</p>
+  </div>
```

Comment:
Please use sematic tag header instead of using div tags for sematic purposes

Reply:
- [ ] Good review @tingamapuro04, however, it would be more helpful to the student to point out the importance of using semantic tags in HTML, in case the student doesn't know what semantic tags are.